.clj-kondo/config.edn

```diff
@@ -706,7 +707,13 @@
 
   printable-namespaces
   {:linters
-   {:discouraged-var {:level :off}}}
+   {:main-without-gen-class {:level :off}
+    :discouraged-var        {clojure.pprint/pprint     {:level :off}
+                             clojure.core/print        {:level :off}
+                             clojure.core/printf       {:level :off}
+                             clojure.core/println      {:level :off}
+                             clojure.string/lower-case {:level :off}
+                             clojure.string/upper-case {:level :off}}}}
```

Comment:
Enables `discouraged-var` for `printable-namespaces`, except for these 6 functions, where it's disabled.